indexer/editable_map_object.cpp

```diff
@@ -352,27 +352,6 @@ void EditableMapObject::SetCuisines(std::vector<std::string> const & cuisines)
 
 void EditableMapObject::SetPointType() { m_geomType = feature::GeomType::Point; }
 
-void EditableMapObject::RemoveBlankAndDuplicationsForDefault()
-{
-  StringUtf8Multilang editedName;
-  string_view defaultName;
-  m_name.GetString(StringUtf8Multilang::kDefaultCode, defaultName);
-
-  m_name.ForEach([&defaultName, &editedName](int8_t langCode, string_view name)
-  {
-    auto const duplicate = langCode != StringUtf8Multilang::kDefaultCode && defaultName == name;
-    if (!name.empty() && !duplicate)
-      editedName.AddString(langCode, name);
-  });
-
-  m_name = editedName;
-}
-
-void EditableMapObject::RemoveNeedlessNames()
-{
-  RemoveBlankAndDuplicationsForDefault();
```

Comment:
Removing blank names is still needed, right? So the code above with !name.empty() should stay? Or can it be done more efficiently in another way? 
CC @kirylkaveryn

Reply:
This method is used to clean up empty names. And used in ios.

Maybe it would be better to set `nil` to the names that shouldn't be added to the list of changes (when a user adds a name but decides to skip this change later and delete translation from the list) and `empty string` when a user cleans up names manually?

Reply:
This is a good question: how tag removal is encoded now in the OM editor core? Is it supported at all? If not, should empty values signal that these keys should be removed from the OSM at the tag merging stage, when there are local tags, and remote tags?